parachain/pallets/basic-channel/rpc/Cargo.toml

```diff
@@ -0,0 +1,52 @@
+[package]
+name = "snowbridge-basic-channel-rpc"
+version = "0.1.0"
+edition = "2021"
+authors = [ "Snowfork <contact@snowfork.com>" ]
+repository = "https://github.com/Snowfork/snowbridge"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { version = "3.0.0", package = "parity-scale-codec", default-features = false, features = [ "derive" ] }
+jsonrpc-core = "18.0.0"
+jsonrpc-core-client = "18.0.0"
+jsonrpc-derive = "18.0.0"
+serde = { version = "1.0.136", features = ["derive"], optional = true }
+
+sp-api = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.19", default-features = false }
```

Comment:
rpc handlers run in the outer node / client, so they are compiled to native code and must the use `std` feature. So you should remove the `default-features = false` to ensure they are linked to std.